api_data.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+"""Py version check"""
+if sys.version_info[0] == 3:
+    from urllib.request import urlopen
+else: 
+	from urllib import urlopen 
+
+def output_cve_id(type=None, name=None, version=None): 
+	"""Take as input, a package name, package version. 
+	Queries cve-search' dataset for any reported 
+	vulnerabilities of the requested package. If 
+	vulnerability exists, outputs cve-id(s).
+	"""
+	if version:
```

Comment:
@kartiksibal nope, you should test the values you get:
Did you get a name and version? --> query for name+version
Did you get only a name? --> query for name
Did you get only a version --> query for version

Eventually you could check if this can be searched more specifically in CPEs?